invokeai/backend/model_manager/probe.py

```diff
@@ -224,7 +224,15 @@ def get_model_type_from_checkpoint(cls, model_path: Path, checkpoint: Optional[C
         ckpt = ckpt.get("state_dict", ckpt)
 
         for key in [str(k) for k in ckpt.keys()]:
-            if key.startswith(("cond_stage_model.", "first_stage_model.", "model.diffusion_model.", "double_blocks.")):
+            if key.startswith(
+                (
+                    "cond_stage_model.",
+                    "first_stage_model.",
+                    "model.diffusion_model.",
+                    "double_blocks.",
+                    "model.diffusion_model.double_blocks.",
```

Comment:
Would be nice to document these magic strings as we add more of them:
```suggestion
                    # FLUX models in the official BFL format contain keys with the "double_blocks." prefix.
                    "double_blocks.",
                    # Some FLUX checkpoint files contain transformer keys prefixed with "model.diffusion_model".
                    # This prefix is typically used to distinguish between multiple models bundled in a single file.
                    "model.diffusion_model.double_blocks.",
```